### ISSUE TITLE
alpine huts should show up earlier

### ISSUE THREAD

**Issue by matkoniecz** [tilezen/vector-datasource#1407]
tl;dr

`alpine_hut` poi min_zoom **`13`** from `15`.

===

* **What did you see?**

http://www.openstreetmap.org/relation/5162163#map=19/49.54372/20.11861 at https://mapzen.com/tangram/play/?#18.25000/49.54396/20.11811 has no indicator that it is a alpine hut or named building

* **What did you expect to see?**

alpine hut data, or at least names of buildings for alpine huts mapped as polygons together with building (this is a correct tagging)

* **What map location are you having problems with?** _City and country are helpful, as well as tile coordinates or latitude / longitude._  
* **Screenshot?** _Props for animated gifs._

See above

default OSM style is displaying tourism=alpine_hut from z13 (very early)

![selection_001](https://user-images.githubusercontent.com/899988/31679521-aaab7922-b371-11e7-89ee-c40fa31b5e53.png)

http://wiki.openstreetmap.org/wiki/Tag:tourism%3Dalpine_hut https://taginfo.openstreetmap.org/tags/tourism=alpine_hut#map is a widely used tag indicating a very important object, rarely existing so data overhead is minimal

https://github.com/tilezen/vector-datasource/search?utf8=%E2%9C%93&q=alpine_hut&type= seems to be querried, so maybe failure is caused by alpine huts mapped as polygons, but https://mapzen.com/tangram/play/?#19.00000/49.60417/18.82297 with http://www.openstreetmap.org/node/718705156#map=17/49.60385/18.82659 mapped as node also failed to appear on my attempt to display alpine_hut nodes

**Comment by nvkelso** [comment]
I think this is "works as expected" now, but with some existing issues you are probably interested in.

How it works now...

The `alpine_hut` tag is stripped off the building and included in the `pois` layer as a point label.

<img width="568" alt="screen shot 2017-10-18 at 12 54 55" src="https://user-images.githubusercontent.com/853051/31739721-b81fc562-b403-11e7-81a3-d00b9b62b44f.png">

That leaves the building polygon not know what kind of building it is, because it's only tagged `building=yes`.

<img width="429" alt="screen shot 2017-10-18 at 12 55 17" src="https://user-images.githubusercontent.com/853051/31739719-b8007a5e-b403-11e7-840f-89c6665b6415.png">

<img width="323" alt="screen shot 2017-10-18 at 12 57 37" src="https://user-images.githubusercontent.com/853051/31739785-f885f8ba-b403-11e7-98a9-d2efe7af8e2e.png">

We could guess / make it consistent with the `pois` logic. That's proposed for the next release under: https://github.com/tilezen/vector-datasource/issues/487

The remaining piece from this issue is proposing to move `alpine_hut` pois to zoom 13 from zoom 15. I agree they should show up earlier so keeping this issue open and changing title.

**Comment by nvkelso** [comment]
Both the **pois** `min_zoom` and the **buildings** `kind_detail` via https://github.com/tilezen/vector-datasource/issues/487 are verified on dev.
